battleship.py

```diff
@@ -0,0 +1,45 @@
+from random import randint
+
+board = []
+
+for x in range(5):
+    board.append(["O"] * 5)
+
+def print_board(board):
+    for row in board:
+        print " ".join(row)
+
+print "Let's play Battleship!"
+print_board(board)
+
+def random_row(board):
+    return randint(0, len(board) - 1)
+
+def random_col(board):
+    return randint(0, len(board[0]) - 1)
+
+ship_row = random_row(board)
+ship_col = random_col(board)
+
+# Everything from here on should go in your for loop!
+# Be sure to indent four spaces!
+for turn in range(4):
```

Comment:
can you think of a way to allow the game to continue until you kill the ship or quit?